css/small-screens.css

```diff
@@ -0,0 +1,58 @@
+#last-news article div img{
+  width: 258px;
+}
+#more-stories > article > div > img {
+  width: 258px;
+}
+
+#part-2 > div:first-child > article:first-child img{
+  height: 100%;
+} 
+#part-2 img{
+  height:100%;
+} 
+
+#part-2 article h3{
+  font-size: 16px;
+  line-height: 16px;
+}
+
+.side-article img {
+  height: 172px;
+  width: 258px;
+
+}
+.part11-header{
+  text-align: center;
+  font-size: 20px;
+}
+.part11-paragraph{
+  text-align: center;
+}
+#part-11 .mag-button{
+  display: block;
+  margin: auto;
```

Comment:
`SUGGESTION`

You can also use the `d-block` and  `m-auto` class name for this selector element and in all other occurrences.
You can also use the screen size prefix for different screen size as shown [here](https://getbootstrap.com/docs/4.0/utilities/spacing/#notation).